data_pipeline_api/file_formats/object_file.py

```diff
@@ -85,6 +85,26 @@ def write_table(file: IOBase, component: str, table: Table):
 UNITS_SUFFIX = "_units"
 
 
+def get_string_array(array) -> np.ndarray:
+    """Attempt to coerce an arbitrary numpy array into a decoded string-type array."""
+    return np.char.decode(np.array(array, dtype=np.string_))
+
+
+def get_string_list(array) -> List[str]:
+    """Attempt to coerce an arbitrary numpy array into a string list."""
+    return list(get_string_array(array))
+
+
+def get_single_string(array) -> str:
+    """Attempt to coerce an arbitrary numpy array into a single string."""
```

Comment:
minor: it's not really arbitrary, it's quite specifically shaped

Reply:
I felt that "attempt" was sufficient cover for all abuses.